4.4.8/Dockerfile.datahub-plugin

```diff
@@ -0,0 +1,21 @@
+# Use the official GeoNetwork v4.4.8 image as the base
+FROM geonetwork:4.4.8
+
+ARG GEONETWORK_VERSION=4.4.8
+ARG PLUGIN_VERSION=${GEONETWORK_VERSION}
+ARG PLUGIN_DOWNLOAD_URL="https://sourceforge.net/projects/geonetwork/files/GeoNetwork_opensource/v${GEONETWORK_VERSION}/gn-datahub-integration-${PLUGIN_VERSION}-0.zip/download"
+
+USER root
+
+RUN apt-get update && \
+    apt-get install -y \
+    wget \
+    unzip && \
```

Comment:
the base image already has `unzip` and uses `curl` for the download so I guess this install step is not necessary?

Reply:
and switching users as well